src/pyargus/models.py

```diff
@@ -52,6 +59,8 @@ def from_json(cls, data: dict) -> Incident:
                 if kwargs["end_time"] != "infinity"
                 else datetime.max
             )
+        elif "end_time" in kwargs:
```

Comment:
The difference between `"end_time" in kwargs` and `kwargs["end_time"]` might not be obvious on a Monday morning/Friday evening.

Reply:
Yeah, I agree; the logic hit me as opaque as I wrote it. This code does, in fact already assume that `'end_time'` is *always* present in `kwargs` (which is only true if the dict is a valid Argus response), so the logic would be the same if it read just:
```suggestion
        else:
```

(but it might be just as opaque)

Reply:
Seeing the whole context I actually think just an else is better. A datetime cannot be falsey after all.